src/tacacs/bash_tacplus/bash_tacplus.c

```diff
@@ -350,40 +350,39 @@ int is_local_user(char *user)
     }
 
     struct passwd pwd;
-    struct passwd *pwdresult;
-    char *buf;
-    size_t bufsize = sysconf(_SC_GETPW_R_SIZE_MAX);
-    if (bufsize == -1) {
-        bufsize = DEFAULT_SC_GETPW_R_SIZE_MAX;
-    }
+    struct passwd *pwp;
```

Comment:
>pwp [](http://example.com/codeflow?start=19&length=3)

Is it so easy to confuse `pwd`. how about `ppwd`?

Reply:
Fixed.